GH_Lettuce/3rd_AutonomousGreenhouseChallenge_2ndExperiment/README.md

```diff
@@ -0,0 +1,15 @@
+# **International Autonomous Greenhouse Challenge - 3rd edition - 2nd experiment**
```

Comment:
Add info about the images data to README.

Remove full data and images.txt files.